### ISSUE TITLE
Feature: Generate JSON filse describing teams

### ISSUE THREAD

**Issue by Cu3PO42** [robocode-dev/robocode-api-bridge#4]
It is my understanding that in order to "correctly" wrap a team, every contained robot needs to be wrapped and additionally, an extra folder plus JSON file should be generated describing the team composition.

The first step is already happening (with some exceptions, see #3), but the latter seems to be missing. I looked at a `.team` file from a JAR and it seems to be a relatively straightforward 1:1 translation that's required here. I'll take a look at implementing this later.

**Comment by flemming-n-larsen** [comment]
@Cu3PO42 
You are right that it should be pretty straight forward. The .team file must be examined the same way as the .properties for a Bot, and mapped into .json file wiht the `teamMembers`. 😊
